File: frontend/src/app/models/Partner.ts
export class Partner {
    id: number;
    alias: string;
    type: string;
    direction:string;
    application: string;
    processFlowType: string;
    description: string;


    constructor(){
        this.id = 0;
        this.alias = '';
        this.type = '';
        this.direction = 'INBOUND';
        this.application = '';
        this.processFlowType = 'MESSAGE';
        this.description = '';
    }

}

File: mq-app/src/main/java/com/gemini/mqapp/model/Partner.java
package com.gemini.mqapp.model;

import com.gemini.mqapp.util.PDirection;
import com.gemini.mqapp.util.ProcessFlowType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Partner {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;
    private String alias;
    private String type;
    private PDirection direction;
    private String application;
    private ProcessFlowType processFlowType;
    private String description;


}
